### ISSUE TITLE
Teammate name

### ISSUE THREAD

**Issue by Skycocoo** [CMU-IDS-2020/CMU-IDS-2020.github.io#3]
Hi,

I found one of my teammates' name is not correct on the website:

```
Team vectorization: Xinwen Liu, Xinyu Lin, Yuxi Luo, Shaobo G Xan
```

Should be 

```
Team vectorization: Xinwen Liu, Xinyu Lin, Yuxi Luo, Shaobo Guan
```

I'm wondering if it is possible to correct this?

**Comment by domoritz** [comment]
Thank you. Fixed.

**Comment by Skycocoo** [comment]
I believe there is one more line that needs to be changed? (https://github.com/CMU-IDS-2020/CMU-IDS-2020.github.io/blob/main/index.html#L42)

**Comment by domoritz** [comment]
Sorry for missing it. Fixed in 5100ff6.
